wqflask/wqflask/snp_browser/snp_browser.py

```diff
@@ -645,6 +643,79 @@ def snp_density_map(self, query, results):
         #for i in range(n_click):
         #    href = url_for('snp_browser', first_run="false", chosen_strains_mouse=self.chosen_strains_mouse, chosen_strains_rat=self.chosen_strains_rat, variant=self.variant_type, species=self.species_name, gene_name=self.gene_name, chr=self.chr, start_mb=self.start_mb, end_mb=self.end_mb, limit_strains=self.limit_strains, domain=self.domain, function=self.function, criteria=self.criteria, score=self.score, diff_alleles=self.diff_alleles)
 
+class SnpPage(object):
+
+    def __init__(self, request):
+        self.request_values = request
+        self.sEcho = self.request_values['sEcho']
+
+        self.snp_browser = SnpBrowser(request)
+        self.rows_count = self.snp_browser.rows_count
+        self.table_rows = self.snp_browser.table_rows
+        self.header_data_names = self.snp_browser.header_data_names
+
+        logger.info(self.table_rows[0])
```

Comment:
**suggestion:** Let's try avoid sprinkling log info. FYI, I'm currently adding a bunch unittests here: https://github.com/genenetwork/genenetwork2/pull/409 so adding log info make things really noisy

Reply:
This was part of my debugging, must have been left out before the commit. It was just for debugging purpose.